### DISCUSSION TITLE
Does anyone has syncWindows examples other than provided in the documentation

### DISCUSSION THREAD

**Discussion by raviusit** [argoproj/argo-cd#8195] [Q&A]
Does anyone has syncWindows examples other than those provided in the documentation (https://argo-cd.readthedocs.io/en/stable/user-guide/sync_windows/) 
I am looking for better real-world examples.

**Comment by leoluz** [comment]
I'm not sure if this is what you are looking for but there are many test cases implemented validating different scenarios while configuring sync windows. I would recommend taking a look at the [TestSyncWindows_CanSync](https://github.com/argoproj/argo-cd/blob/master/pkg/apis/application/v1alpha1/types_test.go#L1550) test as it is more descriptive. But there are many other sync windows related tests in the same file that might help.
